packages/theme/css/foundations/size.css

```diff
@@ -7,7 +7,6 @@
   --salt-size-indicator: 1px;
   --salt-size-selectable: 12px;
   --salt-size-bar-strong: 4px;
-  /* --salt-size-bar-small: 4px; */
```

Comment:
this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable